onnxruntime/core/optimizer/gemm_activation_fusion.cc

```diff
@@ -65,6 +65,12 @@ Status GemmActivationFusion::ApplyImpl(Graph& graph, bool& modified, int graph_l
       continue;
     }
 
+    NodeArg* node_output = node.MutableOutputDefs()[0];
```

Comment:
nit: the new code can be moved to line 58. If data type is not correct, it will stop earlier.